Fix ansible galaxy issues (#2229)
galaxyproject/training-material: topics/admin/tutorials/ansible/tutorial.md

>
>    1. We will call our group `my_hosts`
>

> >    2. Create [an inventory file](https://docs.ansible.com/ansible/2.9/user_guide/intro_inventory.html) with the group `my_hosts` and `localhost ansible_connection=local`, which tells ansible to not use SSH, and just use the local connection. Additionally, you should explicitly set the `ansible_user` to your username on the server. Ansible has changed it's behaviour over time regarding whether or not `ansible_user` is defined, and it's most effective to define it explicitly even when it sometimes can be inferred.

Reviewer: ```suggestion
>    2. Create [an inventory file](https://docs.ansible.com/ansible/2.9/user_guide/intro_inventory.html) with the group `my_hosts` and `localhost ansible_connection=local`, which tells Ansible to not use SSH, and just use the local connection. Additionally, you should explicitly set the `ansible_user` variable to the username to use when connecting to the server. Ansible has changed its behaviour over time regarding whether or not `ansible_user` is defined, and it is most effective to define it explicitly even when it can sometimes be inferred.
```
